File: app/src/main/java/com/sakuna63/tumbin/application/preferences/TokenPrefsSchema.java
package com.sakuna63.tumbin.application.preferences;

import com.rejasupotaro.android.kvs.annotations.Key;
import com.rejasupotaro.android.kvs.annotations.Table;

@Table(name = "token")
public class TokenPrefsSchema {
    @Key(name = "token")
    String token;
    @Key(name = "token_secret")
    String tokenSecret;
}

File: app/src/main/java/com/sakuna63/tumbin/application/preferences/TokenPrefsSchema.kt
package com.sakuna63.tumbin.application.preferences

import com.rejasupotaro.android.kvs.annotations.Key
import com.rejasupotaro.android.kvs.annotations.Table

@Suppress("unused")
@Table(name = "token")
class TokenPrefsSchema {
    @Key(name = "token")
    internal var token: String? = null
    @Key(name = "token_secret")
    internal var tokenSecret: String? = null
}
